src/librustc_codegen_ssa/back/link.rs

```diff
@@ -174,8 +174,10 @@ pub fn get_linker(sess: &Session, linker: &Path, flavor: LinkerFlavor) -> (PathB
     // UWP apps have API restrictions enforced during Store submissions.
     // To comply with the Windows App Certification Kit,
     // MSVC needs to link with the Store versions of the runtime libraries (vcruntime, msvcrt, etc).
+    // Similarly, apps targeting the Games API partition need to link to the OneCore versions of
```

Comment:
See second-to-last paragraph here: https://docs.microsoft.com/en-us/cpp/porting/upgrade-your-code-to-the-universal-crt?view=vs-2019#where-to-find-the-universal-crt-files